src/DataGroup/Element/AdditionalInformation.php

```diff
@@ -30,30 +30,28 @@ class AdditionalInformation implements QrCodeableInterface, SelfValidatableInter
      */
     private $billInformation;
 
-    public function getMessage(): ?string
+    public static function create(
+        ?string $message,
+        ?string $billInformation = null
+    ) : self
```

Comment:
Should it be possible to create object without any data? with only nulls

Reply:
It would not make sense, but it would be allowed. It would eventually be the same behaviour like when no `AdditionalInformation` at all is provided.